changelogs/fragments/38118-postgresql_user-fix_conn_limit_type.yml

```diff
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - conn_limit type is set to 'int'.
```

Comment:
Maybe mention 'postgresql_user' module here? I think the fragments all get merged so it wouldn't be clear which module had changed otherwise.